src/uwloc/data/wav.py

```diff
@@ -1,30 +1,54 @@
 import logging
-from datetime import datetime
+from datetime import datetime, timezone
 from typing import Tuple
 
 import numpy as np
 import numpy.typing as npt
+import pytz
 import scipy.io.wavfile as wf
 import wavinfo
 
 logger = logging.getLogger(__name__)
 
 
+def parse_date(dstr: str) -> datetime:
+    assert "(UTC" in dstr, "Expected a timezone suffix of the form: (UTC-<offset>)"
```

Comment:
Having an assert here is not very clear of what the exception would be... you'd only get `AssertionError`. Maybe be more specific and specify the exception?

```python
if "(UTC" not in dstr:
    raise ValueError("Expected a timezone suffix of the form: (UTC-<offset>)")
```